src/cmake-config/config.h.in

```diff
@@ -23,6 +23,22 @@ constexpr int minor() { return @DRAWPILE_PROTO_MINOR_VERSION@; }
 constexpr int port() { return @DRAWPILE_PROTO_DEFAULT_PORT@; }
 } // namespace proto
 
+namespace setting_default {
```

Comment:
Moving it there is good for now. I was thinking about putting settings all into a data model to make it easier for the UI to react to settings changes, and to make it easier to abort changes to settings, and to make settings persist when they are saved instead of waiting until the application quits. It seems like that could be a good generalisable solution to the problem of where to put this stuff too, since that interface could just not go through QSettings for fixed or non-persistent settings.